xml/deployment_preparation.xml

```diff
@@ -454,7 +465,7 @@ caasp3-stack-admin|10.81.1.51</screen>
   </sect2>
  </sect1>
 
- <sect1 xml:id="sec.deploy.prepare.disk_images">
+ <sect1 xml:id="sec.deploy.preparation.disk_images">
   <title>Installing from Disk Images</title>
```

Comment:
Let's change this headline to "Virtual Disk Images" it's confusing to have ISO Images and Disk Images.